### ISSUE TITLE
Debug

### ISSUE THREAD

**Issue by buiduongthe** [yiicod/yii2-socketio#17]
Dear Yii2cod

Now, I can't debug code function handle in MarkAsReadEvent.php? I want to debug from client to Server Step by Step.
I'm send message from Server to Client for result OK.

Thanks

**Comment by phuongdev89** [comment]
me too, can not handle message from client

**Comment by phuongdev89** [comment]
yeah, I found solution, you should comment https://github.com/yiicod/yii2-socketio/blob/master/Broadcast.php#L71-L72

**Comment by lexxorlov** [comment]
It suppose that you have db on the server side. You can add empty db connection for this or extend this class and comment the lines as proposed above

**Comment by orangeShadow** [comment]
Hi, I am sorry but this solution breaks  package.
```
Yii::$app->db->close();
Yii::$app->db->open();
```
May be you have stable version ? Can I use this package in production ?

**Comment by orangeShadow** [comment]
> It suppose that you have db on the server side. You can add empty db connection for this or extend this class and comment the lines as proposed above

Sorry you can explain me, why I need to reopen mysql connection ? And why in readme has any words about this ?
